analyzer/windows/modules/packages/edge.py

```diff
@@ -10,7 +10,7 @@ class Edge(Package):
     ]
 
     def start(self, url):
-        edge = self.get_path("msedge.exe")
+        edge = self.get_path_glob("msedge.exe")
```

Comment:
Does this work, as there is no glob chars `*` nor `?` in the string?

Reply:
yeah confusingly the glob is in the list of paths to search not in the name provided in the argument